Worktrolly/worktrolly
# Dev/work completion graph

### Functionality:
Implementation of work completion graph

### Solution:
Added work completion pie cart in task cards. Showing the amount of work done.

### Risk level:

### How to test:
- ng serve
- firebase emulators:start
- Go to Board
- Check Task card

ScreenShots:

![WhatsApp Image 2021-02-17 at 23 31 51](https://user-images.githubusercontent.com/54079190/108311730-89e20680-71db-11eb-8c44-a61f96a3f384.jpeg)
![WhatsApp Image 2021-02-17 at 23 31 59](https://user-images.githubusercontent.com/54079190/108311732-8b133380-71db-11eb-9811-881577af09d1.jpeg)
